lunatrace/bsl/backend/src/topic-ingester/schemas/generated-schema-1-validator.ts

```diff
@@ -213,10 +178,11 @@ const typeMap: any = {
         { json: "name", js: "name", typ: "" },
         { json: "summary", js: "summary", typ: "" },
         { json: "language", js: "language", typ: "" },
+        { json: "severity", js: "severity", typ: "" },
```

Comment:
This should be an enum that reflects our db: https://github.com/lunasec-io/lunasec/blob/master/lunatrace/bsl/hasura/migrations/lunatrace/1642199444402_init/up.sql#L186